content/en/docs/instrumentation/ruby/getting-started.md

```diff
@@ -1,112 +1,192 @@
+## Prerequisites
+
+Ensure that you have the following installed locally:
 
 - MRI Ruby >= `3.0`, jruby >= `9.3.2.0`, or truffleruby >= 22.1
-- Docker Compose
+- [Bundler](https://bundler.io/)
+
+{{% alert color="warning" %}} jruby only targets compatibility with MRI Ruby
+2.6.8; which is EOL. This project does not officially support MRI Ruby 2.6.8,
+and provides jruby support on a best-effort basis until the jruby project
+supports compatibility with more modern Ruby runtimes.
+
+truffleruby is tested, but support is best-effort at this time. {{% /alert %}}
+
+## Example Application
+
+The following example uses a basic [Sinatra](https://sinatrarb.com/)
+application. If you are not using Sinatra, that's ok — you can use OpenTelemetry
+Ruby with other web frameworks as well, such as Rails and Rack. For a complete
+list of libraries for supported frameworks, see the
+[regsitry](/ecosystem/registry/?component=instrumentation&language=ruby).
+
+For more elaborate examples, see
+[examples](/docs/instrumentation/ruby/examples/).
+
+### Dependencies
+
+To begin, set up a new Gemfile in a new directory:
+
+```sh
+bundle init
+```
+
+Now install Sinatra and Puma:
+
+```sh
+bundle add sinatra puma
+```
+
+## Create the sample HTTP Server
+
+Create a file `app.rb` and add the following code to it:
+
+```ruby
+require 'rubygems'
+require 'bundler/setup'
+require 'sinatra/base'
+
+class App < Sinatra::Base
+  set :bind, '0.0.0.0'
+  set :port, 8080
 
-> jruby only targets compatibility with MRI Ruby 2.6.8; which is EOL. This
-> project does not officially support MRI Ruby 2.6.8, and provides jruby support
-> on a best-effort basis until the jruby project supports compatibility with
-> more modern Ruby runtimes.
->
-> truffleruby is tested, but support is best-effort at this time.
+  get '/rolldice' do
+    (rand(6) + 1).to_s
+  end
 
-### Installation
+  run! if app_file == $0
+end
+```
+
+Run the application with the following command and open
+<http://localhost:80800/rolldice> in your web browser to ensure it is working.
+
+```sh
+ruby app.rb
+```
+
+## Instrumentation
 
-The first step is to add these gems to your Gemfile:
+Install the `opentelemetry-sdk` and `opentelemetry-instrumentation-all`
+packages:
 
 ```sh
-gem 'opentelemetry-sdk'
-gem 'opentelemetry-exporter-otlp'
-gem 'opentelemetry-instrumentation-all'
+bundle add opentelemetry-sdk opentelemetry-instrumentation-all
 ```
 
 The inclusion of `opentelemetry-instrumentation-all` provides
 [instrumentations][auto] for Rails, Sinatra, several HTTP libraries, and more.
 
-### Initialization
-
 The OpenTelemetry initialization needs to happen early in your application
-lifecycle. For Rails applications, the usual way to initialize OpenTelemetry is
-in a Rails initializer. For other Ruby services, perform this initialization as
-early as possible in the start-up process.
+lifecycle. Perform this initialization as early as possible in the start-up
```

Comment:
My only hesitation here is that the majority of our contributors, maintainers, and support requests are specifically around using OTel with Ruby On Rails. 

I suspect removing references to initializing Rails applications vs using Sinatra will increase our support requests for documentation from our user base. 

I would like to keep the reference to Rails initializers in the documentation in addition to the Sinatra example.

Reply:
I would agree with this. While it's generally good to be as neutral as possible, I think we should keep mentioning Rails specifically because it's just so prominent in the world of Ruby.

Reply:
I initially wanted to build the sample app with rails but Sinatra appeared to be easier and less steps involved. I think there are 2 options:

- Replace sinatra with Rails
- Add a "Note" for the rails specific initalizers

I can do both, I lean towards the first, although it is a little bit more work

Reply:
I think in most cases folks will be retrofitting into existing Rails apps, which is one reason our original docs where intended to be copy/paste modify and we didn't build an entire sample app. 

Of course if you have the bandwidth to put that together it will be most welcome.

I do realize that the goal here is to have a symmetric example, so I think the Sinatra app is pretty good. I really appreciate you adding this example in the docs. Thank you for doing that!

Reply:
I'll try to retrofit it into rails, let me see how far I get :)

Reply:
so... I reworked it and it's now rails, please give it another look, hope the things I did make sense, ruby is not a language I am fluent in ;-)